### ISSUE TITLE
Try to push test coverage on english_language_generator to close to 100%

### ISSUE THREAD

**Issue by gabbard** [isi-vista/adam#601]

**Comment by gabbard** [comment]
@LichMaster98 : This may be a good task for @aakum03 or @jamart28 .  Can you summarize what our current state of coverage is?

**Comment by LichMaster98** [comment]
Our current state of coverage on: https://github.com/isi-vista/adam/pull/604 is all default cases are tested. We currently don't adequately test all the runtime errors in the language generator. To do so, several of the tests will need ontologies defined breaking several of our default assumptions causing the specific failure cases. A non-exhaustive list of examples is: non-3rd person versions of verbs, non-plural indicator of nouns, etc. These edge cases are currently not in our `Phase1Ontology`.
